package.json

```diff
@@ -586,6 +586,7 @@
 		"esbuild": "^0.11.12",
 		"eslint": "^8.13.0",
 		"eslint-config-prettier": "^8.5.0",
+		"eslint-plugin-unused-imports": "^2.0.0",
```

Comment:
Curious, why this instead of https://github.com/import-js/eslint-plugin-import?

Reply:
That one does look pretty cool.  Would be interesting to try that one instead and see what it complains about

Reply:
Does https://github.com/import-js/eslint-plugin-import actually fix unused imports? I've added it and it looks like it's linting a bunch of other stuff, but not that. Maybe I missed it.

Reply:
I took a look and agree with your assessment @phillco.  I did notice that `eslint-plugin-import` has lots of cool rules, tho, so am definitely up for activating it in a follow-up.  In particular, the [`caseSensitive` option](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-unresolved.md#casesensitive) of `no-unresolved` would have saved @AndreasArvidsson some headache on one of his PRs

Reply:
https://github.com/cursorless-dev/cursorless/issues/1146